### PR TITLE
feat: added aurora animation to splash screen

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

Adding a new splash screen background with an animated canvas

## Notice

- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

n/a

## Risk

Pretty low risk, tested on most major browsers.

## Testing

While on the "splash" screen clicking any of the buttons or connecting a wallet, make sure nothing seems laggy or un-usable.

## Screenshots (if applicable)
Screen Shot 2022-06-23 at 11 38 01 AM

### CODE REVIEW

File: src/pages/ConnectWallet/ConnectWallet.tsx [hunk 1]
```diff
@@ -104,98 +102,107 @@ export const ConnectWallet = () => {
...
+              <FoxIcon boxSize='100%' color='white' />
+            </Circle>
+            <Flex
+              flexDir='row'
+              textAlign='center'
+              letterSpacing='-4px'
+              fontSize={{ base: '6xl', lg: '8xl' }}
+              mb={6}
+            >
+              <RawText color='whiteAlpha.400' fontWeight='medium' lineHeight='1'>
```
Comment (0xApotheosis) [inline src/pages/ConnectWallet/ConnectWallet.tsx:124]: The contrast here is a _little_ low.

Screen Shot 2022-06-23 at 10 52 04 am

**Review by 0xApotheosis — COMMENTED** [review]
Nice! One comment on contrast.

File: src/components/AuoraBackground.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
+import { useEffect, useRef } from 'react'
```
Comment (0xdef1cafe) [inline src/components/AuoraBackground.js:1]: this filename has a small typo, and should be `.ts` too

File: src/components/AuoraBackground.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+      animationFrameId = window.requestAnimationFrame(draw)
+    }
+    setup()
+
+    window.addEventListener('resize', resize)
+
+    return () => {
+      window.cancelAnimationFrame(animationFrameId)
+      window.removeEventListener('resize')
```
Comment (0xApotheosis) [inline src/components/AuoraBackground.js:191]: We're missing the `listener` arg here.

**Review by 0xdef1cafe — APPROVED** [review]

**Review by 0xApotheosis — APPROVED** [review]

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**